neurobooth_os/tasks/saccade/saccade_task.py

```diff
@@ -91,7 +92,7 @@ def run_trials(self, prompt=True):
         self.target.size = self.pointer_size_pixel
         self.target.draw()
         self.win.flip()
-        self.update_tablet_background(0, 0)
+        self.Tablet.update_screen(0, 0)
```

Comment:
IDK how this attribute is initialized, but should Tablet be lowercase?

Reply:
This was a mistake, there should've been no Tablet there at all. Fixed it when I was testing on staging.